src/fontMetricsData.js

```diff
@@ -1,1998 +1,1998 @@
+        "69": [0, 0.68333, 0.12028, 0, 0.67833],
+        "70": [0, 0.68333, 0.13305, 0, 0.65277],
+        "71": [0, 0.68333, 0.08722, 0, 0.77361],
+        "72": [0, 0.68333, 0.16389, 0, 0.74333],
+        "73": [0, 0.68333, 0.15806, 0, 0.38555],
+        "74": [0, 0.68333, 0.14028, 0, 0.525],
+        "75": [0, 0.68333, 0.14528, 0, 0.76888],
+        "76": [0, 0.68333, 0, 0, 0.62722],
+        "77": [0, 0.68333, 0.16389, 0, 0.89666],
+        "78": [0, 0.68333, 0.16389, 0, 0.74333],
+        "79": [0, 0.68333, 0.09403, 0, 0.76666],
+        "80": [0, 0.68333, 0.10257, 0, 0.67833],
+        "81": [0.19444, 0.68333, 0.09403, 0, 0.76666],
+        "82": [0, 0.68333, 0.03868, 0, 0.72944],
+        "83": [0, 0.68333, 0.11972, 0, 0.56222],
+        "84": [0, 0.68333, 0.13305, 0, 0.71555],
+        "85": [0, 0.68333, 0.16389, 0, 0.74333],
+        "86": [0, 0.68333, 0.18361, 0, 0.74333],
+        "87": [0, 0.68333, 0.18361, 0, 0.99888],
+        "88": [0, 0.68333, 0.15806, 0, 0.74333],
+        "89": [0, 0.68333, 0.19383, 0, 0.74333],
+        "90": [0, 0.68333, 0.14528, 0, 0.61333],
+        "91": [0.25, 0.75, 0.1875, 0, 0.30667],
+        "93": [0.25, 0.75, 0.10528, 0, 0.30667],
+        "94": [0, 0.69444, 0.06646, 0, 0.51111],
+        "95": [0.31, 0.12056, 0.09208, 0, 0.51111],
+        "97": [0, 0.43056, 0.07671, 0, 0.51111],
+        "98": [0, 0.69444, 0.06312, 0, 0.46],
+        "99": [0, 0.43056, 0.05653, 0, 0.46],
+        "100": [0, 0.69444, 0.10333, 0, 0.51111],
+        "101": [0, 0.43056, 0.07514, 0, 0.46],
+        "102": [0.19444, 0.69444, 0.21194, 0, 0.30667],
+        "103": [0.19444, 0.43056, 0.08847, 0, 0.46],
+        "104": [0, 0.69444, 0.07671, 0, 0.51111],
+        "105": [0, 0.65536, 0.1019, 0, 0.30667],
+        "106": [0.19444, 0.65536, 0.14467, 0, 0.30667],
+        "107": [0, 0.69444, 0.10764, 0, 0.46],
+        "108": [0, 0.69444, 0.10333, 0, 0.25555],
+        "109": [0, 0.43056, 0.07671, 0, 0.81777],
+        "110": [0, 0.43056, 0.07671, 0, 0.56222],
+        "111": [0, 0.43056, 0.06312, 0, 0.51111],
+        "112": [0.19444, 0.43056, 0.06312, 0, 0.51111],
+        "113": [0.19444, 0.43056, 0.08847, 0, 0.46],
+        "114": [0, 0.43056, 0.10764, 0, 0.42166],
+        "115": [0, 0.43056, 0.08208, 0, 0.40889],
+        "116": [0, 0.61508, 0.09486, 0, 0.33222],
+        "117": [0, 0.43056, 0.07671, 0, 0.53666],
+        "118": [0, 0.43056, 0.10764, 0, 0.46],
+        "119": [0, 0.43056, 0.10764, 0, 0.66444],
+        "120": [0, 0.43056, 0.12042, 0, 0.46389],
+        "121": [0.19444, 0.43056, 0.08847, 0, 0.48555],
+        "122": [0, 0.43056, 0.12292, 0, 0.40889],
+        "126": [0.35, 0.31786, 0.11585, 0, 0.51111],
+        "163": [0, 0.69444, 0, 0, 0.76909],
+        "305": [0, 0.43056, 0, 0.02778, 0.32246],
+        "567": [0.19444, 0.43056, 0, 0.08334, 0.38403],
+        "768": [0, 0.69444, 0, 0, 0.51111],
+        "769": [0, 0.69444, 0.09694, 0, 0.51111],
+        "770": [0, 0
```

Comment:
I tried verifying these two characters manually with [opentype.js Glyph Inspector](https://opentype.js.org/glyph-inspector.html).  0.27778 seems like a good approximation to the `advanceWidth` of `!` which is 278 (after dividing by 1000), and ditto for the next few characters.  But I'm confused that the space character (32) has width 0.  It seems to have `advanceWidth` of 250...  Should we fix this now?

Reply:
The glyph inspector shows xMin, yMin, xMax, yMax to be `undefined` for the `space` character so a width and height of 0 for that character makes sense to me.  There is no black mark to measure the width or height of.

I'm not really sure why these metrics have more decimal places then should occur when dividing an integer by 1000.